src/main/java/org/jabref/gui/undo/RedoAction.java

```diff
@@ -21,11 +25,15 @@ public RedoAction(Supplier<LibraryTab> tabSupplier, DialogService dialogService,
         this.tabSupplier = tabSupplier;
         this.dialogService = dialogService;
 
-        // TODO: The old listener should be removed. Otherwise, memory consumption will increase.
-        stateManager.activeTabProperty().addListener((observable, oldValue, activeLibraryTab) -> {
+        ChangeListener<Optional<LibraryTab>> listener = (observable, oldValue, activeLibraryTab) -> {
             activeLibraryTab.ifPresent(libraryTab ->
                     this.executable.bind(libraryTab.getUndoManager().getRedoableProperty()));
-        });
+
```

Comment:
This empty line is not necessary, is it?

Reply:
actually this empty line is for exists to separate lines 30 and 32 from each other. But I can remove.